test/verify/check-storage-nfs

```diff
@@ -189,13 +195,16 @@ class TestStorageNfs(StorageCase):
         self.login_and_go("/storage")
 
         m.execute("mkdir /home/foo /home/bar")
+        self.addCleanup(m.execute, "systemctl restart nfs-server")
         m.write("/etc/exports", "/home/foo 127.0.0.0/24(rw)\n/home/bar 127.0.0.0/24(rw)\n")
+        self.restore_file("/etc/exports")
```

Comment:
Again, write_file(). Please feel free to add and plumb through a new `post_restore_action` argument to write_file() to simplify/unify these cases.